Azael/index.js

```diff
@@ -0,0 +1,64 @@
+//1. NaN
+//Question: Comment your answer in your js file**: What does NaN stand for?
+
+// Answer ==> Stands for NaN "Not a Number". Is a term used in mathematics and computer science to describe a non-numeric value. It may also be a placeholder for an expectednumeric result that cannot be defined as a floating point number.
+
+// Check if "hello" * 1 is NaN.
+
+let hello = "hello";
```

Comment:
You have to multiply "hello" to 1 
Like for example:  console.log(isNaN("hello" *1));